### PR TITLE
DOT-1134 landing page part 2

### PR DESCRIPTION
**Pull request by edwh**
Update to the new design.

### CODE REVIEW

File: app/Services/DiscourseService.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+            foreach ($topics as $topic) {
+                foreach ($categories as $category) {
+                    if ($topic->category_id == $category->id) {
+                        $topic->category = $category;
+                    }
+                }
+            }
+        } catch (\Exception $ex) {
+            Log::error("Error retrieving microtasking discussion topics for username '{$username}': " . $ex->getMessage());
```
Comment (ngm) [inline app/Services/DiscourseService.php:42]: No longer only microtasking related topics.

**Review by ngm — APPROVED** [review]
Noticed that a number of the translations are still using the translatedX computed properties, not the new __() method.  No need to go through and change them all here.. just wondering why? (probably because it was introduced in a commit after this pull request was started, I'm guessing)

**Comment by edwh** [comment]
> Noticed that a number of the translations are still using the translatedX computed properties, not the new __() method. No need to go through and change them all here.. just wondering why? (probably because it was introduced in a commit after this pull request was started, I'm guessing)

Just to be consistent until we do the story to go through and move them over.

File: app/Services/DiscourseService.php (hunk 1 above)
Comment (edwh) [reply app/Services/DiscourseService.php]: Ah yes.

**Pull request merged**